.travis.yml

```diff
@@ -15,188 +15,189 @@ env:
-    - "TESTDIR=Java/undertow-edge"
-    - "TESTDIR=Java/undertow-jersey-c3p0"
-    - "TESTDIR=Java/undertow-jersey-hikaricp"
-    - "TESTDIR=Java/vertx"
-    - "TESTDIR=Java/vertx-web"
-    - "TESTDIR=Java/wicket"
-    - "TESTDIR=Java/beyondj"
-    - "TESTDIR=Java/wildfly-ee7"
-    - "TESTDIR=JavaScript/express"
-    - "TESTDIR=JavaScript/hapi"
-    - "TESTDIR=JavaScript/koa"
-    - "TESTDIR=JavaScript/nodejs"
-    - "TESTDIR=JavaScript/ringojs"
-    - "TESTDIR=JavaScript/ringojs-convenient"
-    - "TESTDIR=JavaScript/sailsjs"
-    - "TESTDIR=Lua/lapis"
-    - "TESTDIR=Lua/openresty"
-    - "TESTDIR=Nim/jester"
-    - "TESTDIR=Nim/nawak"
-    - "TESTDIR=Perl/dancer"
-    - "TESTDIR=Perl/kelp"
-    - "TESTDIR=Perl/mojolicious"
-    - "TESTDIR=Perl/plack"
-    - "TESTDIR=Perl/web-simple"
-    - "TESTDIR=PHP/cakephp"
-    - "TESTDIR=PHP/hhvm"
-    - "TESTDIR=PHP/php"
-    - "TESTDIR=PHP/cygnite-php-framework"
-    - "TESTDIR=PHP/codeigniter"
-    - "TESTDIR=PHP/clancats"
-    - "TESTDIR=PHP/fat-free"
-    - "TESTDIR=PHP/fuel"
-    - "TESTDIR=PHP/kohana"
-    - "TESTDIR=PHP/laravel"
-    - "TESTDIR=PHP/limonade"
-    - "TESTDIR=PHP/lithium"
-    - "TESTDIR=PHP/lumen"
-    - "TESTDIR=PHP/micromvc"
-    - "TESTDIR=PHP/phalcon"
-    - "TESTDIR=PHP/phalcon-micro"
-    - "TESTDIR=PHP/phpixie"
-    - "TESTDIR=PHP/pimf"
-    - "TESTDIR=PHP/silex"
-    - "TESTDIR=PHP/silex-orm"
-    - "TESTDIR=PHP/slim"
-    - "TESTDIR=PHP/symfony2"
-    - "TESTDIR=PHP/symfony2-stripped"
-    - "TESTDIR=PHP/yaf"
-    - "TESTDIR=PHP/yii2"
-    - "TESTDIR=PHP/zend"
-    - "TESTDIR=PHP/zend1"
-    - "TESTDIR=PHP/phreeze"
-    - "TESTDIR=Python/asyncio"
-    - "TESTDIR=Python/bottle"
-    - "TESTDIR=Python/cherrypy"
-    - "TESTDIR=Python/django"
-    - "TESTDIR=Python/falcon"
-    - "TESTDIR=Python/flask"
-    - "TESTDIR=Python/historical"
-    - "TESTDIR=Python/klein"
-    - "TESTDIR=Python/pyramid"
-    - "TESTDIR=Python/tornado"
-    - "TESTDIR=Python/turbogears"
-    - "TESTDIR=Python/uwsgi"
-    - "TESTDIR=Python/web2py"
-    - "TESTDIR=Python/wheezyweb"
-    - "TESTDIR=Python/wsgi"
-    - "TESTDIR=Racket/racket-ws"
-    - "TESTDIR=Ruby/grape"
-    - "TESTDIR=Ruby/ngx_mruby"
-    - "TESTDIR=Ruby/padrino"
-    - "TESTDIR=Ruby/rack"
-    - "TESTDIR=Ruby/rails"
-    - "TESTDIR=Ruby/rails-stripped"
-    - "TESTDIR=Ruby/sinatra"
-    - "TESTDIR=Ruby/sinatra-sequel"
-    - "TESTDIR=Rust/iron"
-    - "TESTDIR=Rust/nickel"
-    - "TESTDIR=Rust/hyper"
-    - "TESTDIR=Scala/akka-http"
-    - "TESTDIR=Scala/colossus"
-    - "TESTDIR=Scala/finagle"
-    - "TESTDIR=Scala/fintrospect"
-    - "TESTDIR=Scala/lift-stateless"
-    - "TESTDIR=Scala/plain"
-    - "TESTDIR=Scala/play2-scala"
-    - "TESTDIR=Scala/scalatra"
-    - "TESTDIR=Scala/scruffy"
-    - "TESTDIR=Scala/spray"
-    - "TESTDIR=Scala/spray-es"
-    - "TESTDIR=Scala/unfiltered"
-    - "TESTDIR=Scala/http4s"
-    - "TESTDIR=Scala/finch"
-    - "TESTDIR=Ur/urweb"
+    ### - "TESTDIR=C/libreactor"
```

Comment:
I believe big part of the changes in this file should be reverted.
You are supposed just to add the new test, not to comment out all others.
I guess you did it just to make Travis build faster while testing.